src/main/java/com/back/global/ai/processor/DetailQuizProcessor.kt

```
@@ -78,42 +70,34 @@ public String buildPrompt() {
-            result = objectMapper.readValue(
-                    cleanedJson,
-                    objectMapper.getTypeFactory().constructCollectionType(List.class, DetailQuizDto.class)
-            );
-        } catch (Exception e) {
-            throw new ServiceException(500, "AI 응답이 JSON 형식이 아닙니다. 응답: " + text);
+        val result: List<DetailQuizDto> = try {
+            objectMapper.readValue(
+                cleanedJson,
+                objectMapper.typeFactory.constructCollectionType(MutableList::class.java, DetailQuizDto::class.java)
+            )
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)